backend/src/atlas_assistant/tools/sql.py

```diff
@@ -202,3 +220,106 @@ def get_prompt(dataset: Dataset) -> str:
 """
 
     return prompt
+
+
+class BarChartDataSchema(BaseModel):
+    type: str
+    value: float | int
+    valueLabel: str
+
+
+class BarChartSchema(BaseModel):
+    title: str
+    categoryField: str
```

Comment:
Because Python, let's use `snake_case` for attribute names. If we _really_ need to serialize differently, let's use an `alias`.